interface/src/components/grid.js

```diff
@@ -89,10 +106,7 @@ class Grid extends Component{
             .style("stroke", "#222")
             .attr("data-legend",function(d) { return d.name})
```

Comment:
This could be a good place for shorthand notation (`=>`) for functions, and elsewhere